### PR TITLE
Docs: Update Canvas videos to mp4 URLs

### PR DESCRIPTION
**Pull request by gguillotte-grafana**
**What this PR does / why we need it**:

grafana/website#10303 changes the embedded videos from MOV format to MP4. Update the embeds accordingly.

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

Blocked by grafana/website#10303

### CODE REVIEW

File: docs/sources/visualizations/canvas.md [hunk 1]
```diff
@@ -16,21 +16,21 @@ weight: 600
...
 Canvas is a new panel that combines the power of Grafana with the flexibility of custom elements. Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.
 
-{{< video-embed src="/static/img/docs/canvas-panel/canvas-beta-overview-9-2-0.mov" max-width="750px" caption="Canvas panel beta overview" >}}
+{{< video-embed src="/static/img/docs/canvas-panel/canvas-beta-overview-9-2-0.mp4" max-width="750px" caption="Canvas panel beta overview" >}}
 
 ## Elements
 
 ### Metric value
 
 The metric value element allows you to easily select the data you want to display on canvas. This element has a unique “edit” mode that can be triggered either through the context menu “Edit” option or by double clicking. When in edit mode you can select which field data that you want to display.
```
Comment (chri2547) [inline docs/sources/visualizations/canvas.md:25]: 'enables' instead of allows.

**Review by chri2547 — APPROVED** [review]
one nit, thanks!

File: docs/sources/visualizations/canvas.md (hunk 1 above)
Comment (gguillotte-grafana) [reply docs/sources/visualizations/canvas.md:25]: IMO this is out of scope for this PR, which is focused solely on supporting grafana/website#10303. The rest of the doc needs an edit pass but this PR isn't the venue.

**Review by nmarrs — APPROVED** [review]

**Comment by gguillotte-grafana** [comment]
This has passed CI and is waiting only on grafana/website#10303.

**Pull request merged**